### ISSUE TITLE
Remove section "component"

### ISSUE THREAD

**Issue by james-nash** [buildit/gravity-ui-web#276]
**Is your feature request related to a problem? Please describe.**
Our pattern library currently contains a `section` "component" under organisms. It's mot really a Gravity component since we don't have any special styling associated with the `<section>`, so I think listing it is a tad confusing. It also obfuscates what (actual) Gravity components the page templates use, because they all use `section`, but configure it to display different things.

I think this `section` thing orignally came about as a bit of a hack to simulate some early website design ideas within the pattern library (and also to work around some Mustache and Pattern Lab limitations/quirks we had at the time), before we were able to implement the corresponding templates in the website codebase.

Long story short, it's clunky, confusing and no longer serves a purpose. I therefore propose we nuke it from orbit.

**Describe the solution you'd like**
* Kill `section` with fire
* Amend sample page templates to directly include or import what they need

**Describe alternatives you've considered**

**Additional context**
